sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 ### Other Changes
 
 - Create separate lists of characters that are allowed within tenant ids and scopes in `AzureCliCredential`.
+- Add logging to `WorkloadIdentityCredential` to help with debugging.
```

Comment:
BTW, if we release GA, we won't need this message, we will be just saying "- Added (support for) `WorkloadIdentityCredential`."

But in this PR, I agree, let's keep it as you have it, and then work on wording as we merge the changes to #5102.